amanuensis/blueprints/filterset.py

```diff
@@ -99,7 +99,7 @@ def update_search(filter_set_id):
         name = flask.request.get_json().get("name", None)
         description = flask.request.get_json().get("description", None)
         filter_object = flask.request.get_json().get("filters", None)
-        graphql_object = flask.request.get_json().get("gqlFilter", {})
+        graphql_object = flask.request.get_json().get("gqlFilter", None)
```

Comment:
this will break something else in other services. I remember setting it '{}' on purpose in the past. 
why did you default the value to None instead of {}?

Reply:
This is set to None for the filter-set update route because we need some way to differentiate between no value being sent and a blank {}. Because when a filter-set is invalid in order to change it back to valid via this endpoint either a graphql_object or a filter-object has to be sent. sending a {} for graphql_object would be considered reseting the filter-set making it valid again. Also the create filter-set endpoint still will add the default {} when creating the filter-set / the server default is {}. Finally if no graphql_object is passed to the update route then nothing changes in the DB.